src/Symfony/Component/Security/Http/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.4
+---
+
+ * [BC BREAK] Removed `InvalidSignatureException` and `MissingClaimException` classes from the `AccessToken\Oidc\Exception` namespace
```

Comment:
I like this PR to cleanup a bit. But having a BC break in 6.4 sounds to be not our plan of having BC breaks only in Major versions. And then I tought about the change ... why is it a BC break? Two `@experimental` Classes are deleted or the Output changes? I guess removing experimental classes in not a bc break by our definition (and as you already said, the Exceptions have been thrown and directly catched, so they never ver available out of the changed code).

Reply:
Changing the public API in a backward incompatible way is a BC break. The fact it is part of an experimental feature only means BC breaks are tolerated as long as we are confident it is improving the feature. See https://symfony.com/doc/current/contributing/code/experimental.html :)

Reply:
Thanks for clearification ;)